components/brave_search_conversion/p3a.cc

```diff
@@ -20,6 +20,12 @@ namespace {
 const char* GetPromoShownPrefName(ConversionType type) {
   switch (type) {
     case ConversionType::kBanner:
+    // Added below four new types just for preventing NOTREACHED().
+    // Need to handle later with p3a team.
+    case ConversionType::kBannerTypeA:
+    case ConversionType::kBannerTypeB:
+    case ConversionType::kBannerTypeC:
+    case ConversionType::kBannerTypeD:
```

Comment:
@DJAndries I introduced another four banner types and `kBannerType` will not be used anymore.
Just added this to prevent NOTREACHED() while testing. Maybe needs some work for p3a support after this PR is merged.